### ISSUE TITLE
No documentation for how to use cylindrical and axisymmetric coordinates

### ISSUE THREAD

**Issue by wilfonba** [MFlowCode/MFC#278]
There is no documentation on how to run simulations using cylindrical and axisymmetric coordinates. There are several intricacies to using cylindrical coordinates (and I assume axisymmetric coordinates) that I've come across when trying to run a 3D pipe flow. Perhaps an additional appendix to the case file documentation would best alleviate this? This isn't a commonly used feature, but it should be documented nonetheless. There should probably be an appendix in the flow visualization documentation detailing how to perform the coordinate transformations correctly as well as it's not as obvious as one might think.

**Comment by sbryngelson** [comment]
Is this still open or no @wilfonba ?

**Comment by wilfonba** [comment]
Still open

**Comment by sbryngelson** [comment]
@ChrisZYJ I cannot remember if you added documentation for this or not. Can you remind me?

**Comment by ChrisZYJ** [comment]
I implemented 2D axisymmetric coordinates and added detailed documentation for it. For the 3D cylindrical coordinates, I wasn't able to get it working (its previous implementation was actually for 2D axisymmetric), so I removed it from the documentation. I intentionally left a few gaps in the Acoustic Supports label numbering to allow for future additions if someone manages to fix the 3D cylindrical coordinates implementation.
